src/services/evm/verify_contract_evm.service.ts

```diff
@@ -62,9 +63,17 @@ export default class VerifyContractEVM extends BullableService {
       const listPromise = [];
       for (let i = 0; i < listRequestVerify.length; i += 1) {
         const requestVerify = listRequestVerify[i];
+        this.logger.info(
+          `Verifying request id ${requestVerify.id} with address ${requestVerify.contract_address}`
+        );
         let codeHash;
+        const compileDetails: any = [];
         try {
-          if (this.isZip(requestVerify.files)) {
+          if (
+            this.isZip(requestVerify.files) ||
```

Comment:
cái này bảo client truyền vào đc ko?

Reply:
validate 1 lần ở client rồi, ở đây mình try catch thôi

Reply:
cho cả FE cả BE cho chắc anh ạ, nhỡ có thằng nghịch linh tinh tại api này đang open :0

Reply:
ý anh là cái type cơ, mà sao bên dưới ko thấy dùng cái files này nhỉ

Reply:
có mà anh, anh expand ra ấy